Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -811,6 +865,47 @@ def check_font_has_a_license(fb, licenses):
 
 @register_test
 @old_style_test(
+    id='com.google.fonts/test/029'
+  , conditions=['license']
+  , priority=CRITICAL
+)
+def check_copyright_entries_match_license(fb, ttFont, license):
+  """Check copyright namerecords match license file"""
+  failed = False
+  for license in ['OFL.txt', 'LICENSE.txt']:
+    if license not in ttFont.reader.file.name:
```

Comment:
Yeah, I know. That was non-sensical indeed.
Please see the aforementioned newest commit addressing this test implementation. I think it is correct now.